File: src/005/p005.ts
import { lcm } from "../utils";

export const solve = (): string => {
  let smallestMultiple = 1;
  for (let i = 2; i <= 20; i++) {
    smallestMultiple = lcm(smallestMultiple, i);
  }
  return smallestMultiple.toString();
};

File: src/005/p005.go
package problem005

import (
	"strconv"

	"github.com/sunilbpandey/project-euler/src/utils/go/math/intutils"
)

func Solve() string {
	lcm := 1
	for n := 2; n <= 20; n++ {
		lcm = intutils.Lcm(lcm, n)
	}
	return strconv.Itoa(lcm)
}
